### PR TITLE
Contact-form

### PR DESCRIPTION
**Pull request by Kumilachew-g**
I have done the following tasks.
   1. Added contact form
   2. Added contact form styling
   3. Added image files
The contact form screenshot:
   
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -255,5 +255,46 @@ <h3>LET’S CONNECT</h3>
...
+      </p>
+      <form action="#" id="comment">
+        <input
+          type="text"
+          id="name"
+          placeholder="Kumilachew"
+          minlength="1"
+          maxlength="30"
+          required
+        />
```
Comment (roshan-bajgain) [inline index.html:274]: - [ ] Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).  The `action` value should be similar to `https://formspree.io/f/{form_id}`. Kindly implement it.

File: index.html [hunk 2]
```diff
@@ -255,5 +255,46 @@ <h3>LET’S CONNECT</h3>
...
+          cols="30"
+          rows="10"
+        ></textarea>
+        <button type="submit" class="btn">Get In Touch</button>
+      </form>
+
+      <div class="ending">
+        <img src="images/Shape@2x.svg" alt="shape" />
+      </div>
+    </footer>
```
Comment (roshan-bajgain) [inline index.html:298]: - [ ] Your contact form is scrolling horizontally kindly fix it
### Your design
hiss

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Kumilachew-g, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -255,5 +255,46 @@ <h3>LET’S CONNECT</h3>
...
+    <!-- Contact Form -->
+    <footer>
+      <div class="contact-form"></div>
+      <h2 class="contact-me">Contact me</h2>
+      <p class="para-text">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      <form action="https://formspree.io/f/mbjwvgbk" id="comment">
```
Comment (Ugboaja-Uchechi) [inline index.html:267]: - [ ] Great job adding the `action` attribute 👍 . But, like the previous reviewer mentioned you need to also add the `method` attribute. Without the `method` attribute we get this error when submitting the form 🙏 .

image

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @Kumilachew-g, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights

- README is professional 👍 
- Linters are passing ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [OPTIONAL] **This is meant to be an important part of your project but, it was missed by the previous reviewer. Kindly make sure your partner commits to your branch in any pair programming task 🙏 .**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (rloterh) [reply index.html]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Kumilachew,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- Form action and method properly implemented

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**